src/theme.ts

```diff
@@ -596,6 +602,37 @@ export const theme = createTheme({
 					color: color.text.danger.value,
 					borderColor: color.border.danger.value,
 				},
+				startIcon: {
+					' > :nth-of-type(1)': {
+						fontSize: '14px',
+					},
+				},
+				endIcon: {
+					' > :nth-of-type(1)': {
+						fontSize: '14px',
+					},
+				},
```

Comment:
Why aren't we applying the font size to all icons regardless of location in the button? Not saying we should, just trying to understand why

Reply:
I don't think I can target the element easily @myarmolinsky, there is no general `.MuiButton-icon` class. Or am I missing something?

Reply:
> I just realized actually, why do we need `> :nth-of-type` at all? Shouldn't we only ever have 1 icon in `startIcon` or `endIcon`?

I'm following MUI's CSS rule, I figure there's a good reason they're targeting it like that

Reply:
> I would expect the icon font size to just match the font size for the rest of the button tbh. Wouldn't make sense to have a 14px icon when our button text is 20px 😕

although it's currently the case this is not necessarily true @myarmolinsky, there are valid reasons to have smaller icon sizes than the text (for example if we switch to a library with sharper/bolder icons). That's why I created different tokens for icon sizes (see the typography PR).

I'll revisit this bit though, I agree the duplication of the rule is not ideal :)